src/codegen/compile.ml

```diff
@@ -943,6 +943,24 @@ module RTS = struct
     E.add_func_import env "rts" "region_store_byte" [I32Type; I64Type; I32Type] [];
     E.add_func_import env "rts" "region_next_id" [] [I32Type];
     E.add_func_import env "rts" "region_meta_loglines" [] [];
+    E.add_func_import env "rts" "region0_size" [I32Type] [I64Type];
+    E.add_func_import env "rts" "region0_grow" [I32Type; I64Type] [I64Type];
+    E.add_func_import env "rts" "region0_load_blob" [I32Type; I32Type; I32Type] [I32Type];
+    E.add_func_import env "rts" "region0_store_blob" [I32Type; I32Type; I32Type] [];
+    E.add_func_import env "rts" "region0_load_byte" [I32Type; I64Type] [I32Type];
+    E.add_func_import env "rts" "region0_store_byte" [I32Type; I64Type; I32Type] [];
+    E.add_func_import env "rts" "region0_load_nat16" [I32Type; I64Type] [I32Type];
+    E.add_func_import env "rts" "region0_store_nat16" [I32Type; I64Type; I32Type] [];
+    E.add_func_import env "rts" "region0_load_nat32" [I32Type; I64Type] [I32Type];
+    E.add_func_import env "rts" "region0_store_nat32" [I32Type; I64Type; I32Type] [];
+    E.add_func_import env "rts" "region0_load_nat64" [I32Type; I64Type] [I64Type];
+    E.add_func_import env "rts" "region0_store_nat64" [I32Type; I64Type; I64Type] [];
+    E.add_func_import env "rts" "region0_load_int16" [I32Type; I64Type] [I32Type];
+    E.add_func_import env "rts" "region0_store_int16" [I32Type; I64Type; I32Type] [];
+    E.add_func_import env "rts" "region0_load_int32" [I32Type; I64Type] [I32Type];
+    E.add_func_import env "rts" "region0_store_int32" [I32Type; I64Type; I32Type] [];
+    E.add_func_import env "rts" "region0_load_int64" [I32Type; I64Type] [I64Type];
+    E.add_func_import env "rts" "region0_store_int64" [I32Type; I64Type; I64Type] [];
```

Comment:
You could probably get away with removing half the functions and using `load/store_wordXX` for both unsigned and signed NatXX/IntXX. The existing implementation does something similar - search for StableMem.load_word16/StableMem.store_word16 for an example (in compile.ml).

Reply:
(they are used to implement both Int16 and Nat16 variants)

Reply:
Did this change.